src/api/apiPaths.ts

```diff
@@ -19,7 +19,7 @@ export enum CristinApiPath {
   Person = '/cristin/person',
   PersonIdentityNumber = '/cristin/person/identityNumber',
   Position = '/cristin/position',
-  Project = '/cristin/project',
+  Project = '/cristin-np-45525-add-facets-to-project-query/project',
```

Comment:
Må gå tilbake i denne før vi merger til develop
```suggestion
  Project = '/cristin/project',
```

Reply:
https://github.com/BIBSYSDEV/nva-cristin-service/pull/335

Reply:
Ruller bare ut dette med riktig path i dev, så vil de vises så snart det er støtte for dette i backend